datadog/resource_datadog_dashboard.go

```diff
@@ -4713,7 +4713,27 @@ func getApmLogNetworkRumSecurityQuerySchema() *schema.Schema {
 				},
 				"compute": {
```

Comment:
Since `compute` and `multi_compute` are mutually exclusive (as per the docs below), should we mark them with `ConflictsWith`? (see it's doc under https://godoc.org/github.com/hashicorp/terraform-plugin-sdk/helper/schema#Schema)

Reply:
Hmm, actually after re-reading this I'm not sure. Are they mutually exclusive or can they coexist?

Reply:
AFAIK you can't use `ConflictsWith` instead lists or maps.

Reply:
Ah, that's right. So we're basically relying on the API to return the proper error message if both are used, right?

Reply:
Yeah I added a TODO below, but I need to change 5 different functions to propagate errors :/